docs/core/compatibility/toc.yml

```diff
@@ -126,6 +126,8 @@ items:
         href: sdk/8.0/msbuild-language.md
       - name: Runtime-specific apps not self-contained
         href: sdk/8.0/runtimespecific-app-default.md
+      - name: "Warning for IsTrimmable with netstandard libraries"
```

Comment:
Please also add an entry to line 1469 (one group is by area, one is by version).